uint8 COMMAND_START = 0 # Start executing actions.
uint8 COMMAND_HALT  = 1 # Stop all executions.
uint8 command

string strategy_name # Used with COMMAND_START only.
---
# No response.